about/package-scope.md

```diff
@@ -18,21 +18,257 @@ Currently, the packages that pyOpenSci reviews also need to fall into the
+may be interested in working with. These data may also be generated from
+scientific workflows or exported from instruments and wearables.
+
+* Examples: [physcraper](https://github.com/pyOpenSci/software-submission/issues/26)
+
+
+### Data deposition
+
+Tools for depositing data into scientific research repositories.
+
+* Examples: [This is an example from rOpenSci - eml](https://github.com/ropensci/software-review/issues/80)
+
+### Data validation and testing:
+
+Tools that enable automated validation and checking of data quality and
+completeness. These tools should be able to support scientific workflows.
+
+* Example: [pandera](https://github.com/pyOpenSci/software-submission/issues/12)
+
+### Scientific software wrappers
+
+Scientific software wrappers refer to packages that provide a Python interface
+for existing scientific packages written in other languages.
+
+These packages should have a clear scientific application. Wrappers must provide
+significant added value to the scientific ecosystem be it in data handling, or
+improved installation processes for Python users.
+
+We strongly encourage submissions that wrap tools that are open-source with
+an OSI-approved license. Exceptions will be evaluated case-by-case,
+considering whether open-source options exist.
+
+<!-- TODO: need an example for this category -->
+* Examples: We don't have a package in this category yet - *Could be your package?*
+
+### Workflow automation & versioning
+Tools that automate and link together workflows and as such support
+reproducible workflows. These
+tools may include build systems and tools to manage continuous integration.
+This also includes tools that support version control.
+
+<!-- TODO: marting git package is a good example but it's needs to be sunsetted as it's no longer maintained. hamilton is just starting review but would fit
+well here  - the link below for it is a presubmission not a review   -->
+* Examples: [Hamilton - currently under review](https://github.com/pyOpenSci/software-submission/issues/74), martin's git package (that is no longer maintained.)
+
+### Citation management and bibliometrics:
+
+Tools that facilitate managing references, such as for writing manuscripts,
+creating CVs or otherwise attributing scientific contributions, or accessing,
+manipulating or otherwise working with bibliometric data. (Example: [Example from rOpenSci - RefManageR](https://github.com/ropensci/software-review/issues/119))
+
+### Data visualization & analysis
+These are packages that enhance a scientists experience visualizing and
+analyzing data.
+
+* Examples: [PyGMT - (also spatial and data munging)](https://github.com/pyOpenSci/software-submission/issues/43),
+
+### Database software bindings
+
+ Bindings and wrappers for database APIs.
+
+ * Example: [Example from rOpenSci - rrlite](https://github.com/ropensci/software-review/issues/6)
+
+
+## Domain areas
+
+In addition, our scope includes focused domain areas. These areas are based on
```

Comment:
The "computational library" class, described here, seems to be missing from the previous section, when it's arguably the first category included?

Reply:
Say a little bit more about what you mean?  
I think you're suggesting that there should be another category above that's like "general library or framework for a domain or research area"?
E.g. [python-graphblas](https://github.com/pyOpenSci/software-submission/issues/81) would be such a library (in addition to being a wrapper)

Reply:
ugh... i missed a few comments here! i will make sure these are added in a separate PR. i am not sure how i missed this